Documentation/RunningOnRaspberryPi.md

```diff
@@ -48,6 +48,13 @@ This step is needed because the original firmware files need to be present on th
 
 Please follow one of the existing guides (for example [here](https://scribles.net/setting-up-serial-communication-between-raspberry-pi-and-pc)) and make sure UART is working on Raspberry Pi OS before proceeding.
 
+If you're using a Raspberry Pi 4 and want to test if the UART is working correctly, you need to do a few extra steps.
```

Comment:
I think this i sonly for a 4B. The normal 4 doesn't have bt.

(i.e. s/4/4B` here)